package.json

```diff
@@ -27,7 +27,7 @@
   ],
   "dependencies": {
     "backbone": "1.2.3",
-    "cartodb.js": "CartoDB/cartodb.js#f0bd380",
+    "cartodb.js": "CartoDB/cartodb.js#5aa87cf",
```

Comment:
Considering cartodb.js v4 is not versioned and things are changing fast, we have experienced that this repo failed on occasions due to incompatibilities, so we changed back to using "commit-based" versioning for the time-being, to make sure things are always working.